### ISSUE TITLE
 Release 4.0 run - 'ExecutionResult' of 3.2.2 is not supported

### ISSUE THREAD

**Issue by Or-Kedmi** [robotframework/robotframework#3888]
Using service 'A', on which version 3.2.2 is installed, I use the following Python code to identify the status of a suite (PASS / FAIL) run by another service 'B' (where the version number should be dynamic and flexible):

`import robot.api`
...
`result = robot.api.ExecutionResult(path_to_output_file)`
`return result.suite.status`

The problem I faced with is when version 4.0 was installed on 'B', service 'A' failed to extract the status from the output.xml file.
In other words, 'ExecutionResult' initialization is not supported in advanced versions:

![2021_03_15_11_45_00_cccc py_Visual_Studio_Code_Administrator_](https://user-images.githubusercontent.com/52378734/111134630-7892dc80-8584-11eb-930f-f49b64f5e32a.png)

I don't want to upgrade 'A' to version 4.0, and on the other hand support version 4.0 for 'B'. If so, how should I identify the run status?

Thanks.

**Comment by pekkaklarck** [comment]
Unfortunately RF 3.2 cannot process output.xml files generated by RF 4.0. It works the other way around, though. You basically need to update 'A' to use RF 4.0 or make sure 'B' uses RF 3.2.
